1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -91,27 +91,28 @@ function generate_kubernetes_config() {
       <name>openshift</name>
       <templates>
         <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
-          <name>agent</name>
-          <image>jenkinsci/jnlp-slave</image>
+          <name>maven</name>
+          <image>openshift/jenkins-slave-maven-centos7</image>
           <privileged>false</privileged>
           <command></command>
           <args></args>
+          <remoteFs>/home/jenkins</remoteFs>
           <instanceCap>2147483647</instanceCap>
-          <label>agent</label>
+          <label>maven</label>
           <volumes/>
           <envVars/>
           <nodeSelector/>
           <remoteFs>/tmp</remoteFs>
         </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
         <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
-          <name>redhatdistortion-maven</name>
-          <image>docker.io/redhatdistortion/maven-jenkins-slave</image>
+          <name>nodejs</name>
```

Comment:
right, this section is defining a new pod template which uses the new openshift/jenkins-slave-nodejs-centos7 image (which is introduced by this PR).  The label/name of "nodejs" means you can reference this pod template for slave-launching purposes by using the "nodejs" label name in your jenkinsfile/job configuration.

However you've made me realize this this hardcoded config means we're going to use the centos slave image even when we're running a rhel master, which is not what we want, so i need to rework this logic a little to be more customized.